packages/playwright-core/src/server/recorder/python.ts

```diff
@@ -150,7 +155,13 @@ export class PythonLanguageGenerator implements LanguageGenerator {
 
   generateHeader(options: LanguageGeneratorOptions): string {
     const formatter = new PythonFormatter();
-    if (this._isAsync) {
+    if (this._isTest) {
+      formatter.add(`
```

Comment:
Let's also configure device emulation by overriding `browser_context_args` fixture, similar to javascript? [Here is an example](https://playwright.dev/python/docs/test-runners#device-emulation).

Reply:
@dgozman I find it tricky to create the curly braces for dictionaries and the double indentation (8 spaces). Any advice.

<img width="712" alt="image" src="https://user-images.githubusercontent.com/2573537/166549854-c52dff42-874f-48f9-bb6e-97dbf0ffe16f.png">

Reply:
Yeah, that's indeed tricky. Let me merge this PR and I can follow up, because it might require some internal refactoring to make that work.